docs/migrations/web/MIGRATION-v3.md

```diff
@@ -175,6 +190,85 @@ Instead of using `.Button--square`, use a `.Button--symmetrical`.
 
 - `.Button--square` → `.Button--symmetrical`
 
+### Button: The `inverted` modifier removed
+
+Button `inverted` modifier was removed. Use themes to switch the button colors.
+
+### Field Group: Alignment
```

Comment:
```suggestion
### FieldGroup: Alignment
```

The component is called `FieldGroup` IMHO.

Reply:
I think it depends on the context. I believe we generally use the Title Case syntax (with space) in the docs and demos, and PascalCase in code.

Reply:
We use Title Case syntax in design too 🤷

Reply:
> It is way easier to address the components with the same name as in the code.

It is (or can be), but we need to do this consciously and throughout the whole design system. It's a change that exceeds the scope of this PR.

Reply:
Ok, let's discuss this on a Long Topics.